go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/dreadl0ck/gopcap v0.0.0-20190327214342-be2a4d78505f
-	github.com/open-traffic-generator/snappi/gosnappi v0.10.4
+	github.com/open-traffic-generator/snappi/gosnappi v0.10.5
```

Comment:
whenever upgrading gosnappi, please do it like so locally: `go get package_name@version` so that go.sum gets updated as well.

please push again after doing it.